objects/00aa5c/script.lua

```diff
@@ -250,20 +252,3 @@ function ToggleAspect(_, _, alt_click)
         })
     end
 end
------
-function upCast(obj)
```

Comment:
Note this function is actually different than the one in `global-script.lua` so something will need to be adjusted.

Reply:
it's very close to the same thing, the only difference is the size parameter